From 7878393176cd60360ea0f6340da85b482de1218b Mon Sep 17 00:00:00 2001
From: Sinthu Raja <sinthu.raja@ti.com>
Date: Mon, 26 Feb 2024 11:53:37 +0000
Subject: [tiL6.12 PATCH v2 9/9] TI: HACK: arm: multi_v7_defconfig: enable
 rpmsg proto driver

Enable rpmsg proto driver as module needed for IPC applications based on
sockets.

Signed-off-by: Sinthu Raja <sinthu.raja@ti.com>
Signed-off-by: Beleswar Padhi <b-padhi@ti.com>
---
 arch/arm/configs/multi_v7_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/configs/multi_v7_defconfig b/arch/arm/configs/multi_v7_defconfig
index befd4bf09672..d66eb1639070 100644
--- a/arch/arm/configs/multi_v7_defconfig
+++ b/arch/arm/configs/multi_v7_defconfig
@@ -1124,6 +1124,7 @@ CONFIG_QCOM_SYSMON=m
 CONFIG_QCOM_WCNSS_PIL=m
 CONFIG_ST_REMOTEPROC=m
 CONFIG_RPMSG_CHAR=m
+CONFIG_RPMSG_PROTO=m
 CONFIG_RPMSG_CTRL=m
 CONFIG_RPMSG_QCOM_SMD=y
 CONFIG_RPMSG_VIRTIO=m
-- 
2.34.1

